### PR TITLE
Add files associated with Docker topic (docker branch)

### PR DESCRIPTION
**Pull request by H1mansh0**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.11.4
+
+WORKDIR /app
+
+RUN pip install flask
+RUN pip install psycopg2-binary
+
+COPY . .
+
+CMD ["flask", "run", "--host=0.0.0.0"]
```
Comment (maksymloktionov) [inline Dockerfile:10]: Let's remove this piece as we don't need it

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.11.4
+
+WORKDIR /app
+
+RUN pip install flask
```
Comment (maksymloktionov) [inline Dockerfile:5]: Please move the libraries to the separate requirements.txt file and add 
RUN python -m pip install -r /requirements-dev.txt in the Dockerfile

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    ports:
+      - "8000:5000"
+    depends_on:
+      - db
+    environment:
+      - POSTGRES_HOST=db
+      - POSTGRES_DB=my_first  
+      - POSTGRES_USER=vlad    
+      - POSTGRES_PASSWORD=1234 
+      - FLASK_APP=app.py # app.py will be needed
```
Comment (maksymloktionov) [inline docker-compose.yaml:13]: We can remove this for now

**Comment by H1mansh0** [comment]
New version added

**Review by maksymloktionov — APPROVED** [review]

**Comment by maksymloktionov** [comment]
Looks good, please squash and merge it

**Pull request merged**